code/game/gamemodes/malfunction/Malf_Modules.dm

```diff
@@ -1,5 +1,21 @@
 #define DEFAULT_DOOMSDAY_TIMER 4500
 
+GLOBAL_LIST_INIT(blacklisted_malf_machines, typecacheof(list(
+		/obj/machinery/field/containment,
+		/obj/machinery/power/supermatter_shard,
+		/obj/machinery/dominator,
+		/obj/machinery/doomsday_device,
+		/obj/machinery/nuclearbomb,
+		/obj/machinery/nuclearbomb/selfdestruct,
+		/obj/machinery/nuclearbomb/syndicate,
+		/obj/machinery/syndicatebomb,
```

Comment:
"<span class='notice'>You tap into the station's powernet. Click on a machine to detonate it, or use the ability again to cancel.</span>""

Per the code, you hack into devices connected to the powernet and overload their circuits to cause them to explode. Bombs aren't connected to the powernet, are they? (I was honestly surprised to even find out they weren't /obj/)
Likewise, having bombs come to life animated seems rather silly.